structuredevent-service-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/rest/controller/CDPStructuredEventV1Controller.java

```diff
@@ -55,4 +71,10 @@ private Response getAuditEventsZipResponse(Collection<CDPStructuredNotificationE
         String fileName = String.format("audit-%s.zip", resourceType);
         return Response.ok(streamingOutput).header("content-disposition", String.format("attachment; filename = %s", fileName)).build();
     }
+
+    private void checkPermissions(Collection<? extends CDPStructuredEvent> events) {
+        Set<String> eventResourceCrns = events.stream().map(event -> event.getOperation().getResourceCrn()).collect(toSet());
+        permissionCheckingUtils.checkPermissionForUserOnResources(STRUCTURED_EVENTS_READ, ThreadBasedUserCrnProvider.getUserCrn(), eventResourceCrns);
```

Comment:
it seems this `STRUCTURED_EVENTS_READ` action is not defined in UMS, possibly it is not used

Reply:
I suggest to check permission similarly to the method in AuditEvent controller